inc/class-admin.php

```diff
@@ -48,6 +48,7 @@ public function setup() {
 		add_filter( 'post_row_actions', [ $this, 'add_api_link_to_posts' ], 10, 2 );
 		add_filter( 'page_row_actions', [ $this, 'add_api_link_to_posts' ], 10, 2 );
 		add_filter( 'admin_bar_menu', [ $this, 'add_api_link_to_admin_bar' ], 999 );
+		add_filter( 'admin_bar_menu', [ $this, 'add_registered_components_link_to_admin_bar' ], 999 );
```

Comment:
Updated to use action.

It's been a long time since this was written, but my guess is I wanted to ensure it was always the last thing, irregardless of what other plugins were doing. Updated to 99.